tests/testthat/test-lazy_profile.R

```diff
@@ -0,0 +1,41 @@
+test_that("can modify lazy profile settings", {
+  #some way to check if .pr$LazyFrame$optimization_toggle works
```

Comment:
```suggestion
  # TODO: some way to check if .pr$LazyFrame$optimization_toggle works
```

Reply:
will be possible from rust-polars 0.31.0